tests/test_toggle.py

```diff
@@ -336,6 +358,38 @@ def test_dynamic_to_json(self):
         self.assertEqual(json.loads(json.dumps(data)), golden)
 
 
+class ToggleLoadTC(unittest.TestCase):
+
+    def test_load(self):
+        fixture = '''[{"fixed": {"show_axis": false}},
+{"dynamic": {"apps": {"euler1d": {"use_sub": false}}}}]'''
+        tg = modmesh.toggle.load(
+            fixture,
+            toggle_instance=modmesh.Toggle.instance.clone())
+        self.assertEqual(tg.apps.euler1d.use_sub, False)
+
+        fixture = '''[{"fixed": {"show_axis": false}},
+{"dynamic": {"apps": {"euler1d": {"use_sub": true}}}}]'''
+        tg = modmesh.toggle.load(
+            fixture,
+            toggle_instance=modmesh.Toggle.instance.clone())
+        self.assertEqual(tg.apps.euler1d.use_sub, True)
+
+    @unittest.skip("the lifecycle issue may cause segfault")
+    def test_load_bad_lifecycle(self):
+        fixture = '''[{"fixed": {"show_axis": false}},
+{"dynamic": {"apps": {"euler1d": {"use_sub": false}}}}]'''
+        # TODO: Need to fix this later. It may be wrong lifecycle handling with
+        # WrapHierarchicalToggleAccess.
+        with self.assertRaisesRegex(
+                AttributeError,
+                r'Cannot get non-existing key "apps.euler1d"'
+        ):
+            modmesh.toggle.load(
+                fixture,
+                toggle_instance=modmesh.Toggle.instance.clone()).apps.euler1d
```

Comment:
`(temporary).apps` works.  But `(temporary).apps.euler1d` crashes or fails.

I am guessing `HierarchicalToggleAccess` does not hold the lifecycle of the parent `Toggle` object.  I tried to use return value policy but it does not help.  I need more time to study how to deal with it.